### PR TITLE
Fix #3841: Combine time and quantity plurals

### LINKED ISSUE #3841: 'time_ago' & quantity plurals should be combined
After #3795, TextViewBindingAdapters combines two strings (time_ago & plurals) into a single one. This ought to just be different plural options so that translators have better context when translating the full conceptual string.

### PR DESCRIPTION
**Pull request by aayushimathur6**
## Explanation
Fixes #3841 

This PR involves Combining plurals and quantity strings from:
` <string name="time_ago">%s ago</string>`

and

```
<plurals name="minutes">
  <item quantity="one">a minute</item>
  <item quantity="other">%s minutes</item>
</plurals>
```

to

```
 <plurals name="minutes_ago">
    <item quantity="one">a minute ago</item>
    <item quantity="other">%s minutes ago</item>
  </plurals>
```

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

| BEFORE  | AFTER |
| ------------- | ------------- |
| Before| after  |
| arabicbefore |afterarabic |

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -481,17 +481,18 @@
   <string name="up_button_disabled">Up</string>
   <string name="down_button_disabled">Down</string>
   <string name="profile_last_visited">%s %s</string>
-  <plurals name="minutes">
-    <item quantity="one">a minute</item>
-    <item quantity="other">%s minutes</item>
+  <plurals name="minutes_ago">
+    <item quantity="zero">zero minutes ago</item>
```
Comment (adhiamboperes) [inline app/src/main/res/values/strings.xml]: This could be the digit "0 minutes ago"

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @aayushimathur6! 

Some feedback:

- [ ] There are some failing tests  in `TextViewBindingAdaptersTest` as a result of the change. Please run the test class locally to find the failing tests or [Interpret the CI results](https://github.com/oppia/oppia-android/wiki/Interpreting-CI-Results). 
- [ ] Please update the PR title as instructed in the essential checklist.

**Comment by aayushimathur6** [comment]
> Thanks @aayushimathur6!
> 
> Some feedback:
> 
> * [ ]  There are some failing tests  in `TextViewBindingAdaptersTest` as a result of the change. Please run the test class locally to find the failing tests or [Interpret the CI results](https://github.com/oppia/oppia-android/wiki/Interpreting-CI-Results).
> * [ ]  Please update the PR title as instructed in the essential checklist.

updated

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (aayushimathur6) [reply app/src/main/res/values/strings.xml]: done

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -468,7 +468,7 @@
   <!-- TextViewBindingAdapters -->
   <string name="just_now">just now</string>
   <string name="last_logged_in_recently">recently</string>
-  <string name="time_ago">%s ago</string>
```
Comment (adhiamboperes) [inline app/src/main/res/values/strings.xml:471]: This string is no longer needed, please delete it.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -481,17 +481,18 @@
   <string name="up_button_disabled">Up</string>
   <string name="down_button_disabled">Down</string>
   <string name="profile_last_visited">%s %s</string>
-  <plurals name="minutes">
-    <item quantity="one">a minute</item>
-    <item quantity="other">%s minutes</item>
+  <plurals name="minutes_ago">
+    <item quantity="zero">0 minute ago</item>
```
Comment (adhiamboperes) [inline app/src/main/res/values/strings.xml]:
```suggestion
    <item quantity="zero">0 minutes ago</item>
```

**Review by adhiamboperes — CHANGES_REQUESTED** [review]
Thanks @aayushimathur6.

This solution is still incomplete since the `fun getPluralString` is still accepting 2 seperate stings. Since the `time_ago` string is nolonger needed or used, you need to refactor `getPluralString` so that where we are currently passing `time_ago` we replace it with `pluralsResId`.

At the same time, you need to remove the TODO comment in that method so that the TODO check will not fail.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (aayushimathur6) [reply app/src/main/res/values/strings.xml]: done

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (aayushimathur6) [reply app/src/main/res/values/strings.xml:471]: done

**Comment by aayushimathur6** [comment]
> Thanks @aayushimathur6.
> 
> This solution is still incomplete since the `fun getPluralString` is still accepting 2 seperate stings. Since the `time_ago` string is nolonger needed or used, you need to refactor `getPluralString` so that where we are currently passing `time_ago` we replace it with `pluralsResId`.
> 
> At the same time, you need to remove the TODO comment in that method so that the TODO check will not fail.

done

**Review by adhiamboperes — COMMENTED** [review]
Thanks @aayushimathur6, this looks good to me.

You are missing the screenshots that prove that the solution works. Can you add before(from develop) and after(from this branch) screenshots? They should show that the text displays correctly still, for both LTR and RTL.

**Comment by aayushimathur6** [comment]
> Thanks @aayushimathur6, this looks good to me.
> 
> You are missing the screenshots that prove that the solution works. Can you add before(from develop) and after(from this branch) screenshots? They should show that the text displays correctly still, for both LTR and RTL.

done

**Comment by aayushimathur6** [comment]
> Hi @aayushimathur6, it looks like some changes were requested on this pull request by @adhiamboperes. PTAL. Thanks!



> Thanks @aayushimathur6.
> 
> This solution is still incomplete since the `fun getPluralString` is still accepting 2 seperate stings. Since the `time_ago` string is nolonger needed or used, you need to refactor `getPluralString` so that where we are currently passing `time_ago` we replace it with `pluralsResId`.
> 
> At the same time, you need to remove the TODO comment in that method so that the TODO check will not fail.

done

**Review by adhiamboperes — APPROVED** [review]
LGTM, thanks @aayushimathur6!

**Pull request merged**